core/sqf/sql/scripts/dcscheck

```diff
@@ -171,17 +174,65 @@ if ( [ $sq_stat == 0 ] || [ $sq_stat == 1 ] ); then
    fi
    echo
   fi
-  echo -e "Process\t\tConfigured\tActual\t\tDown"
-  echo -e "---------\t----------\t------\t\t----"
-  echo -e "DcsMaster\t$cfg_dcsmaster_cnt\t\t$actual_dcsmaster_cnt\t\t$down_dcsmaster_cnt"
-  echo -e "DcsServer\t$cfg_dcsserver_cnt\t\t$actual_dcsserver_cnt\t\t$down_dcsserver_cnt"
-  echo -e "mxosrvr\t\t$cfg_mxo_cnt\t\t$act_mxo_cnt\t\t$down_mxo_cnt\n"
  else
    echo "DCS is not installed. Please install and configure DCS..."
    exit 1
  fi
 else
  echo "Trafodion is not started or is not operational..."
  echo
+ exit 1
+fi
+
+if [ -z "$OPT_VALUE" ]; then
+
+    echo -e "Process\t\tConfigured\tActual\t\tDown"
+    echo -e "---------\t----------\t------\t\t----"
+    echo -e "DcsMaster\t$cfg_dcsmaster_cnt\t\t$actual_dcsmaster_cnt\t\t$down_dcsmaster_cnt"
+    echo -e "DcsServer\t$cfg_dcsserver_cnt\t\t$actual_dcsserver_cnt\t\t$down_dcsserver_cnt"
+    echo -e "mxosrvr\t\t$cfg_mxo_cnt\t\t$act_mxo_cnt\t\t$down_mxo_cnt\n"
+    exit 0
+else
+ echo
+    case $OPT_VALUE in
+        -m)
+            if [ -e $SQ_PDSH ]; then
+                STAT_CMD="$SQ_PDSH -w $activeMaster $jstatcmd -gc "
```

Comment:
How will this work? Don't we need to specify the pid of dcsmaster to get the statistics?

Reply:
Sorry . Yes I noticed that change and forgot to delete my comment